pkg/operation/botanist/controlplane/kubecontrollermanager/kube_controller_manager.go

```diff
@@ -298,6 +298,15 @@ func (k *kubeControllerManager) Deploy(ctx context.Context) error {
 		vpa.Spec.UpdatePolicy = &autoscalingv1beta2.PodUpdatePolicy{
 			UpdateMode: &vpaUpdateMode,
 		}
+		vpa.Spec.ResourcePolicy = &autoscalingv1beta2.PodResourcePolicy{
+			ContainerPolicies: []autoscalingv1beta2.ContainerResourcePolicy{{
+				ContainerName: kubeControllerManagerContainerName,
+				MinAllowed: corev1.ResourceList{
+					corev1.ResourceCPU:    resource.MustParse("20m"),
```

Comment:
I have not seen CPU usage go below `23m` on a cluster with low activity. Does it make sense to set min CPU to `25m` instead of `20m`?

![image](https://user-images.githubusercontent.com/32157254/97475224-fad28980-1972-11eb-8d89-0c9bf3e0de00.png)

Reply:
Yep, done. Thanks for checking in detail!